src/include/daos_cont.h

```diff
@@ -705,6 +705,19 @@ int
 daos_cont_get_perms(daos_prop_t *cont_prop, uid_t uid, gid_t *gids, size_t nr_gids,
 		    uint64_t *perms);
 
+/**
+ * Dump object ID table (OIT) for the snapshot
+ *
+ * \param[in]	coh	Container handle
+ * \param[out]	epoch	epoch of snapshot
+ * \param[in]	name	Optional null terminated name for snapshot.
+ * \param[in]	ev	Completion event, it is optional and can be NULL.
+ *			The function will run in blocking mode if \a ev is NULL.
+ */
+int
+daos_cont_snap_oit_dump(daos_handle_t coh, daos_epoch_t epoch, char *name,
```

Comment:
not sure a use is going to understand what this API means. As a use i don't know what dumping the oit is supposed to do and why i would use it..

Reply:
maybe rename the function to something like: daos_cont_oit_create ?